app/views/stats/subscriptions.html.erb

```diff
@@ -4,4 +4,28 @@
 </div>
 <div class="col-md-8 col-md-offset-1">
     <h2>Subscriptions</h2>
-</div>
+ 	<table class="table inline-grid">
+		<tr>
+			<thead>
+				<th>Subscriptions</th>
+				<th>Tags</th>
+			</thead>
+		</tr>
+		<% @tags.each do |map| %>
+		<% map.each do |subscriptions,tags| %>
+		<tr>
+			<tbody>
+				<td><%=subscriptions%></td>
```

Comment:
Here we could put this, because we'd want to expand back out to the full ranges (since we divided by 10 earlier) -- try this out:

```suggestion
				<td><% (range - 1) * 10 - <%= range*10 %></td>
```

Reply:
Yes makes perfect sense. I figured this `<%=range*10%> - <%=range*10 + 9%>` displays from the minimum range to the range + 9 just so we have it kind of like this:

![count](https://user-images.githubusercontent.com/17080976/54805619-8c9eee80-4c88-11e9-8787-4088efef28fa.png)

is this okay?